tools/aws_cli.py

```diff
@@ -155,6 +156,17 @@ def StartAnalysisVm(args: 'argparse.Namespace') -> None:
         'Created key pair {0:s} in AWS. Your private key is saved in: '
         '{1:s}'.format(key_name, path))
 
+  # Load up the startup script
+  startup_script = None
+  if args.launch_script:
+    startup_script = utils.ReadStartupScript(args.launch_script)
+  else:
+    startup_script = utils.ReadStartupScript(utils.FORENSICS_STARTUP_SCRIPT)
+    # Install ec2-instance-connect to allow SSH connections from the browser.
+    startup_script = startup_script.replace(
```

Comment:
I see. This is for the CLI tool, though. What happens to the forensics package w.r.t. `ec2-instance-connect`? Shouldn't what's below also be surfaced in that package?

Reply:
The end goal here is to be able to support different launch scripts. Perhaps the best answer is to have a forensics package script for each aws, gcp and azure, and we can include `ec2-instance-connect` specifically in the aws one.

Reply:
Sure, but now if you call `aws.forensics.StartAnalysisVm`, in its current state `userData` is `None` and hence `ec2-instance-connect` won't be installed, which is a change in behaviour compared to what CFU did before. This is problematic because `ssh_key_name` is also `None` by default, hence the resulting VM will not be SSHable into if neither a key or the package is provided. What you added in `aws_cli.py` should also be added to `forensics.py` to keep the behaviour we had before but provide the choice of specifying a different startup script.

Reply:
Righto, readjusted to have multiple startup scripts for each provider, and `aws.forensics.StartAnalysisVm` has a default value for the userdata file being the existing AWS one. This should achieve no unexpected changes to anyone directly invoking that method, and allow custom scripts to be provided.